packages/cli/src/cog/cutline.ts

```diff
@@ -11,6 +11,12 @@ const { intersection, union } = pc;
 /** Padding to always apply to image boundies */
 const PixelPadding = 100;
 
+const SourceSmoothScale = 1 + 1e-8;
+
+function copyPoint(p: number[]): [number, number] {
+    return [p[0], p[1]];
```

Comment:
do you expect points to have more than two values in it ? aka why this over `[].slice()`

Reply:
faster.  2ns vs 6ns